src/main/java/com/avispl/symphony/dal/communicator/aggregator/ZoomRoomsAggregatorCommunicator.java

```diff
@@ -1465,9 +1465,16 @@ private void processPaginatedResponse(String url, int pageSize, PaginatedRespons
      * @throws Exception if any error occurs
      */
     private void populateDeviceDetails(String roomId) throws Exception {
-        AggregatedDevice aggregatedZoomRoomDevice = aggregatedDevices.get(roomId);
+            System.out.println("Fetching room details for device " + roomId);

```

Comment:
Do we really need 'System.out.println' here, or should we replace it with a logger?
Please ignore if you find it appropriate

Reply:
Oh right, thank you, this must be removed :)